aqua/diagnostics/histogram/plot_histogram.py

```diff
@@ -110,13 +110,12 @@ def set_ref_label(self):
 
     def set_title(self):
         """Set the title for the plot."""
-        title = "PDF " if self.density else "Histogram "
+        # Use long_name as-is (already contains "Pdf of" or "Histogram of" from aqua.core.histogram)
+        title = self.long_name or self.standard_name or self.short_name or ""
```

Comment:
It would be nice here to make sure that the first letter of the title is capitalized, so that if the long_name is "histogram of..." then "Histogram of " is printed